deps/v8/include/v8.h

```diff
@@ -5878,6 +5878,26 @@ enum class PropertyHandlerFlags {
 };
 
```

Comment:
@TimothyGu As someone not super familiar with the internals of this can you explain a bit about how this is fixing the linked to issues.